dev/docker/eth-dev.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.2'
+
+# creates a nucypher node and a ganache server which can be used for
+# blockchain simulations
+
+# docker-compose -f eth-dev.yml up
+
+# in another shell
+# docker-compose -f eth-dev.yml run nucypher-dev nucypher-deploy contracts --poa --provider-uri http://ganache:7545
+# docker-compose -f eth-dev.yml run nucypher-dev nucypher felix init --checksum-address "<from output>" --network devnet
+# docker-compose -f eth-dev.yml run  nucypher-dev nucypher felix createdb --checksum-address "<from output>" --network devnet
+# docker-compose -f eth-dev.yml run --service-ports nucypher-dev nucypher felix run --checksum-address "<from output>" --network devnet
+
+services:
+  nucypher-dev:
+    ports:
+      - "6151:6151"
+    volumes:
+      - ../..:/code
+    build:
+      context: ../..
+      dockerfile: dev/docker/Dockerfile
+    image: dev:nucypher
+    container_name: nucypher-dev
+    environment:
+      - NUCYPHER_CONFIG_ROOT=/code/config-data
+      - NUCYPHER_FELIX_DB_SECRET=1h918hd98327hf8723f8723f872g38f7g
+  ganache:
+    image: trufflesuite/ganache-cli:v6.1.8
+    ports:
+      - 8545:8545
+    entrypoint:
+      - node
+      - ./build/cli.node.js
+      - --host=0.0.0.0
+      - --port=8545
+      - --gasLimit=7992181
```

Comment:
Heh - that's oddly specific.

Reply:
I copied and pasted it from the ganache docker example.

Reply:
Fair enough.  Perhaps just make it a nice round 8 million.